index.html

```diff
@@ -74,13 +74,8 @@ <h1>
     </section>
 
     <section id="portfolio" class="work">
-        <!----------------------------------->
-        <!-- HTML USED TO FILL UP THIS SPACE -->
-        <!----------------------------------->
     </section>
-    <!--  -->
-    <!-- Modal Pops Up Here -->
-
+    
```

Comment:
- [ ] Kindly make sure you follow the template design as said in the requirements, your popup window should look and behave like the template one.  Check the screencast below for more info 

https://drive.google.com/file/d/1saa5u6zYLIYZTwNyslwcFXBw2FiEbSXs/view?usp=sharing